src/gateway/router/routes/sequencerRoute.ts

```diff
@@ -267,24 +266,15 @@ function prepareTags(
   return { contractTag, inputTag, requestVrfTag, internalWrites, decodedTags, tags, vrfData };
 }
 
-async function compress(transaction: Transaction) {
-  const stringifiedTx = JSON.stringify(transaction);
-  const gzipPromisified = callbackToPromise(gzip);
-  const gzippedData = await gzipPromisified(stringifiedTx);
-
-  return gzippedData;
-}
-
 async function uploadToBundlr(
   transaction: Transaction,
   bundlr: Bundlr,
   tags: GQLTagInterface[],
   logger: RedStoneLogger
 ) {
   const uploadBenchmark = Benchmark.measure();
-  const gzippedData = await compress(transaction);
 
-  const bTx = bundlr.createTransaction(gzippedData, { tags });
+  const bTx = bundlr.createTransaction(JSON.stringify(transaction), { tags });
```

Comment:
shouldn't the tag `{ name: 'Sequencer-Compression', value: 'gzip' },` be also removed from here?